### DISCUSSION TITLE
How to enable bluetooth on macbook 12 retina (2015) with this guide

### DISCUSSION THREAD

**Discussion by bazarov94** [community/community#148181] [New to GitHub]
Hello everyone.
I installed Ubuntu on my 2015 Macbook. Everything works except audio and bluetooth.
To get the bluetooth working I found this guide on github, but being a novice I can't get it working.
https://github.com/leifliddy/macbook12-bluetooth-driver

Would anyone be able to give me some tips?

**Comment by ghostinhershell** [comment]
Hi @bazarov94,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [Apple Support Community](https://discussions.apple.com/welcome). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
